packages/fluentui/accessibility/src/behaviors/index.ts

```diff
@@ -146,3 +146,7 @@ export { default as videoBehavior } from './Video/videoBehavior';
 export * from './Video/videoBehavior';
 export { default as hiddenComponentBehavior } from './Common/hiddenComponentBehavior';
 export { default as cardSelectableBehavior } from './Card/cardSelectableBehavior';
+export { default as datepickerBehavior } from './Datepicker/datepickerBehavior';
+export * from './Datepicker/datepickerBehavior';
```

Comment:
Please don't forget to add new behaviors to packages/fluentui/accessibility/test/behaviors/behavior-test.tsx

Reply:
Please search for "cardBehavior" for inspiration